delta/rdf/src/test/resources/context.json

```diff
@@ -4,9 +4,7 @@
     "@vocab": "http://example.com/",
     "@base": "http://nexus.example.com/",
     "xsd": "http://www.w3.org/2001/XMLSchema#",
-    "xsd2": "http://www.w3.org/2001/XMLSchema#",
     "Person": "http://schema.org/Person",
-    "Person2": "http://schema.org/Person",
```

Comment:
I still need to check that again but it does not take the same prefix when 2 prefixes point to the same target.
It should not happen in real life though.